src/components/Customer/CartDialog.vue

```diff
@@ -28,16 +31,109 @@
             </v-col>
           </v-row>
         </v-container>
+        <v-container v-if="items !== null && items.length > 0">
+          <v-row>
+            <v-col lg="12">
+              <v-card flat>
+                <v-card-title class="display-1 orangeText">Checkout Details</v-card-title>
+              </v-card>
+              <v-divider></v-divider>
+            </v-col>
+            <v-col lg="12">
+              <v-radio-group row v-model="isRecent">
+                <v-radio label="Enter New Address" :value="false"></v-radio>
+                <v-radio label="Choose Recent Address" :value="true"></v-radio>
+              </v-radio-group>
+            </v-col>
+            <template v-if="!isRecent">
+              <v-col lg="12">
+                <v-autocomplete
+                  v-model="selectedAddress"
+                  :search-input.sync="addressSearch"
+                  color="orange"
+                  dense
+                  filled
+                  hide-details
+                  hide-no-data
+                  disable-lookup
+                  label="Enter Address"
+                  solo-inverted
+                  :items="possibleAddresses"
+                ></v-autocomplete>
+              </v-col>
+              <v-col lg="6">
+                <v-text-field
+                  color="orange"
+                  v-model="unit"
+                  dense
+                  filled
+                  label="Enter Unit#"
+                  maxlength="10"
+                ></v-text-field>
+              </v-col>
+              <v-col lg="6">
+                <v-text-field color="orange" v-model="postalCode" label="Postal Code" disabled></v-text-field>
+              </v-col>
+            </template>
+            <v-col lg="12" v-else>
+              <v-radio-group v-model="selectedRecent">
+                <v-radio
+                  v-for="recentAddress in recentAddresses"
+                  :label="recentAddress.address + ' ' + recentAddress.postal + ' #' + recentAddress.unit"
+                  :key="recentAddress.postal"
+                  :value="recentAddress"
+                ></v-radio>
+              </v-radio-group>
+            </v-col>
+            <v-col lg="6">
+              <v-select
+                v-model="promo"
+                :items="['promo1']"
+                label="Apply promotions?"
+                outlined
+                dense
+                color="orange"
+              ></v-select>
+            </v-col>
+            <v-col lg="6">
+              <v-checkbox
+                color="orange"
+                label="Use points to wave delivery fee?"
+                v-if="points >= 300"
```

Comment:
just to check, the points that the customer have is visible in same page right?

Reply:
Yea I think I should display the net points